### ISSUE TITLE
TXT record returns two values - doesn't seem that should be possible

### ISSUE THREAD

**Issue by bbct** [acme-dns/acme-dns#353]
My renewal was failing, the value returned for my TXT record didn't match. 
I used nslookup to see what the value is.
Somehow I've managed to get 2 values stored for the same TXT record:

Non-authoritative answer:
90103513-A497-46F6-944e-32CDf9D25794.My.domain.COM       text =

        "SqTGI30-hNDOzuDCCARZx8_ca8dbhCCJ45emjGEQTec"
90103513-A497-46F6-944e-32CDf9D25794.My.domain.COM       text =

        "M4Js9Ps56wQZn5v6_j45LQLk4ZquUjfxsXiKfRx6gBI"
>

The second one is the correct one, the first one must be older?  Should two TXT records even exist like this? 
Question 1: any idea how I managed to do this?
Question 2: how do I delete the TXT records to start from scratch?

Disclosure - As I am testing this out first, I may have registered this same domain a second time under different credentials, perhaps that is why it is returning two values?. 

Perhaps I should just start with a fresh db, and start over?  I've registered only a couple domains. If I need to start over, what's the best way?

Thank you for any help you can provide.

**Comment by bbct** [comment]
FYI - using Sqlite3, I queried the txt table, there were two rows for each Subdomain. I deleted the oldest for each pair, and it seems to be working now. 
Still not sure how I got two TXT records for each subdomain...

**Comment by aduzsardi** [comment]
afaik , that's the desired behaivor (having 2 txt records) for wildcard certificates

**Comment by bbct** [comment]
Interesting, aduzsardi.  It wasn't actually a wildcard cert I requested, though.  Just a single domain.

**Comment by joohoi** [comment]
Yeah, this is true. An usecase where the CA requires two different tokens
in the same subdomain is a wildcard certificate and the apex domain in the
same certificate.

That said, the CA will respect a correct validation token in any of the the
records. Additional ones do not matter.

**Comment by hvisage** [comment]
Now, if @joohoi could be so kind as to allow the txt_limit PR to solve the problems for acme.sh using a single account for multiple domainnames in the same request
